### DISCUSSION TITLE
video recording problem

### DISCUSSION THREAD

**Discussion by Sigmalalala** [bonsai-rx/bonsai#1179] [Q&A]
I use this workflow to record a video for a certain time long such as 1 minutes. The video i got is 1  minutes long indeed, but i found that the actual recording time is not  1 minutes.  The video was only recorded for 58 seconds, but it was stretched to 1 minute. The fps of my industrial camera varies all the time. How can i solve this problem? Thank you so much if you can help me.
![1670595247049(1)](https://user-images.githubusercontent.com/118099060/206721521-fcde1cc6-abfe-4c80-9f69-ccb84c3df28b.jpg)

**Comment by bruno-f-cruz** [comment]
We had an identical discussion a while back in #1112.

**Reply by Sigmalalala** [reply]
Thank you very much for your reply. I've seen this before. So there is no effective solution yet, right?
